jwst/superbias/bias_sub.py

```diff
@@ -12,25 +10,20 @@
 
 def do_correction(input_model, bias_model):
     """
-    Short Summary
-    -------------
-    Execute all tasks for Super-Bias Subtraction
+    Execute all tasks for Super-Bias Subtraction.
 
     Parameters
     ----------
-    input_model: data model object
-        science data to be corrected
-
-    bias_model: super-bias model object
-        bias data
+    input_model : stdatamodels.jwst.datamodels.ramp.RampModel
+        Science data to be corrected
+    bias_model : sustdatamodels.jwst.datamodels.superbias.SuperBiasModel
+        Bias data
```

Comment:
I think the model name is sufficient here, and in the rest of these changes.

```suggestion
    input_model : RampModel
        Science data to be corrected
    bias_model : SuperBiasModel
        Bias data
```